packages/orchestrator-ui-components/src/pages/subscriptions/WFOSubscriptionDetailPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SubscriptionProvider, TreeProvider } from '../../contexts';
+import { WFOSubscription } from '../../components';
+import { useRouter } from 'next/router';
+
+export const WFOSubscriptionDetailPage = () => {
+    const router = useRouter();
+    const { subscriptionId } = router.query;
+
+    return (
+        <SubscriptionProvider>
+            <TreeProvider>
+                {subscriptionId && (
+                    <WFOSubscription
+                        subscriptionId={
```

Comment:
Without knowing the details, this kind of code (checking on array) breaks of course every typing system, but is also a bit vague. So subcriptionId can be either a scalar or an array?

Reply:
True, but unfortunately the type of subscriptionId is actually any of the 2 types: 
![image](https://github.com/workfloworchestrator/orchestrator-ui/assets/20791917/283c2ebb-607c-4da4-99b9-400eced473f5)

Reply:
Ok, wouldn't then make sense to isolate this 'ugliness' and just create a function that always returns a string | undefined? And can it indeed be undefined?